iree_tests/benchmarks/sdxl/benchmark_sdxl_rocm.py

```diff
@@ -172,6 +176,9 @@ def test_sdxl_rocm_benchmark(goldentime_rocm_e2e, goldentime_rocm_unet,
     job_summary_lines.append(mean_line)
     logging.getLogger().info(mean_line)
 
+    # new line
+    job_summary_lines.append("\n")
```

Comment:
drop trivial comment (or add more context like "New line separator between result sections.")
```suggestion
    job_summary_lines.append("\n")
```